transformation/src/test/scala/org/broadinstitute/monster/encode/transformation/TransformationPipelineSpec.scala

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.monster.encode.transformation
+
+import better.files.File
+import org.broadinstitute.monster.common.PipelineBuilderSpec
+
+class TransformationPipelineSpec extends PipelineBuilderSpec[Args] {
+  behavior of "TransformationPipeline"
+
+  private val testFileLocation = s"${File.currentWorkingDirectory}/src/test/test-files"
+  private val truthDir = File.currentWorkingDirectory / "src" / "test" / "test-files" / "outputs"
+  private val compareDir = File.currentWorkingDirectory / "src" / "test" / "test-files" / "outputs-to-compare"
+  private val compareDirString = compareDir.pathAsString
+  private val inputDirString = s"${testFileLocation}/inputs"
+
+  override val testArgs =
+    Args(inputPrefix = inputDirString, outputPrefix = compareDirString)
+
+  override val builder = EncodeTransformationPipelineBuilder
+
+  /**
+    *
+    * Helper method to call the parsing method on the truth-files and the files-to-test.
+    *
+    * @param subDir The sub-directory of the outputs dir containing the files to read
+    * @return A tuple of Set of Json, where the first one is the Set-to-test and the second one is the truth-Set
+    */
+  private def compareTruthAndCompSets(subDir: String): Unit = {
+    it should s"have written the correct $subDir data" in {
+      val expected = readMsgs(truthDir / subDir)
+      val actual = readMsgs(compareDir / subDir)
+      actual should contain theSameElementsAs expected
+    }
+  }
+
+  private val outputDirs = Set(
+    "human_donor"
+  )
+
+  outputDirs.foreach {
+    it should behave like compareTruthAndCompSets(_)
```

Comment:
Yay testing! So these two records in /test/test-files/inputs/ are for the unit test, right?
I think I need to incorporate the comparison method you've implemented to the encode-extract integration test, too. Good work!

Reply:
Yes, they are! I was mostly able to base this off of the existing ClinVar unit tests, but it's definitely a helpful tool and was really easy to use.